sequencer/src/lib.rs

```diff
@@ -326,16 +325,31 @@ pub async fn init_node(
     let priv_key = priv_keys[config.node_index as usize].clone();
     let enc_key = KeyPair::generate(&mut StdRng::seed_from_u64(config.node_index));
 
-    // Wait for other nodes to connect.
-    while !networking.run_ready() {
-        let connected = networking.get_connected_client_count().await;
-        tracing::info!(
-            "waiting for start signal ({}/{} connected)",
-            connected,
-            config.config.total_nodes,
-        );
-        sleep(Duration::from_secs(1)).await;
-    }
+    // // Wait for other nodes to connect.
+    orchestrator_client
+        .wait_for_all_nodes_ready(node_index.into())
+        .await;
+    let wait_time = Duration::from_secs(1);
```

Comment:
What is wait_time used for?

Reply:
The wait time in between polls (replaces the previous `sleep` statement)

Reply:
So to be clear, this sleep is only used for connecting to the server, not during consensus? I'm just thinking 1 second seems a little slow for consensus messages but totally fine for startup-related stuff

Reply:
Ah actually this is the wait time in between sequencer -> web server polls, so I guess 1 second would be too slow. How about 100 ms?

Reply:
That sounds more reasonable, let's double check with the consensus team

Reply:
Yes sounds like 100 ms is good, long term we should consider making this configurable